composer.lock

```diff
@@ -1583,7 +1583,7 @@
                 "logging",
                 "psr-3"
             ],
-            "time": "2019-11-13T10:00:05+00:00"
+            "time": "2019-12-20T14:15:16+00:00"
```

Comment:
is time really needed here?

Reply:
doesn't make sense to update every time u update release if not needed?

Reply:
`composer.lock` is an autogenerated file when we install/update dependencies. We are not manually updating it.

Reply:
ok.